### DISCUSSION TITLE
Cant see all wallpapers in app

### DISCUSSION THREAD

**Discussion by Abbas-15** [rocksdanister/lively#2459] [Q&A]
I can only see about 12 base wallpapers on the app. I want to know if there are more available because I think there are (Looking at the preview pictures) and if so, how do I access them?

**Comment by rocksdanister** [comment]
https://www.reddit.com/r/LivelyWallpaper/comments/kfvc2o/how_to_download_wallpapers_from_lively_deviantart/

https://github.com/rocksdanister/lively/wiki/Sample-Wallpaper-Projects
